Solutions/Infoblox Cloud Data Connector/Data Connectors/InfobloxCloudDataConnector.json

```diff
@@ -59,8 +68,7 @@
                 "scope": "Workspace",
                 "requiredPermissions": {
                     "read": true,
-                    "write": true,
-                    "delete": true
+                    "write": true

```

Comment:
Please add delete permission also. Because of that validation is failing. Thanks

Reply:
Done